common/modal.js

```diff
@@ -347,11 +348,14 @@
             vm.identities.push(session.client.identities[i]);
         }
 
-
         for(var i = 0; i<session.attributes.length; i++){
-            if(session.attributes[i].display_name && session.attributes[i].display_name !== user.display_name){
-                session.attributes[i].id = session.attributes[i].id.substring(24);
-                vm.groupList.push(session.attributes[i]);
+            if(session.attributes[i].display_name && session.attributes[i].display_name !== user.display_name && vm.identities.indexOf(session.attributes[i].id) == -1){
+                if (session.attributes[i].id.indexOf("globus") > -1) {
```

Comment:
Since the next line is looking for the exact position of "24",  you could change this check to look for the whole prefix instead so

```javascript
session.attributes[i].id.indexOf("https://auth.globus.org/") === 0
```